From: Tobias Heider <me@tobhe.de>
Date: Tue, 24 Dec 2024 00:18:56 +0100
Subject: Fix target directory path, it contains the toolchain in our case

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1610b4f..2204c5f 100644
--- a/Makefile
+++ b/Makefile
@@ -92,7 +92,7 @@ ifeq ($(SEV),1)
 	mv target/release/libkrun.so target/release/$(KRUN_BASE_$(OS))
 endif
 ifeq ($(OS),Linux)
-	patchelf --set-soname $(KRUN_SONAME_$(OS)) --output $(LIBRARY_RELEASE_$(OS)) target/release/$(KRUN_BASE_$(OS))
+	patchelf --set-soname $(KRUN_SONAME_$(OS)) --output $(LIBRARY_RELEASE_$(OS)) target/*/release/$(KRUN_BASE_$(OS))
 else
 ifeq ($(EFI),1)
 ifeq ($(OS),Darwin)
